actions_build/index.js

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable*/
```

Comment:
I have Disabled Lint beacuse while commiting `eslint *.js ./spec/*.js ./lib/*.js actions/**/*.js --fix` runs causing all auto fixable problems to be fixed.